tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -308,8 +340,11 @@ int main() {
     test_parse_chrono_format_specs<char>();
     test_parse_chrono_format_specs<wchar_t>();
 
-    // #ifndef __clang__ // TRANSITION, LLVM-48606
-    //     test_day_formatter<char>();
-    //     test_day_formatter<wchar_t>();
-    // #endif // __clang__
+#ifndef __clang__ // TRANSITION, LLVM-48606

```

Comment:
This shouldn't be necessary because we aren't trying to validate `constexpr` here.